docs/user/site.rst

```diff
@@ -124,6 +124,7 @@ wifi24 \: optional
     configure any subset of them:
 
     - ``ap`` creates a master interface where clients may connect
+    - ``owe`` creates a OWE interface where clients may connect (requires ``ap`` to be present)
```

Comment:
Does it? Even if no transition mode was enabled?

Reply:
Currently yes, as it's state is set implicitly from the state of the regular client network.

This is something which should be improved, but I'm not quite sure how to handle this in the web wizard, as the settings required to change is dependent whether or not we are using transition mode.

Reply:
Altered to allow any combination.